tests/foreman/ui/test_contenthost.py

```diff
@@ -76,6 +79,16 @@ def setUpClass(cls):
             environment=cls.env,
             organization=cls.session_org,
         ).create()
+        if not settings.cdn:
+            # if not using cdn by settings, have to force uploading the
+            # manifest and attach a subscription to activation key
```

Comment:
I think we have extra arg like `force_manifest_upload` for `setup_org_for_a_rh_repo` which should handle that. And if it's not adding subscription to ak in the end - then it's a flaw in its implementation and it should be updated rather than extra workaround added before each `setup_org_for_a_rh_repo` call. Your thoughts?